drafts/2019-11-19-this-week-in-rust.md

```diff
@@ -48,46 +48,43 @@ If you are a Rust project owner and are looking for contributors, please submit
+260 pull requests were [merged in the last week][merged]
+
+[merged]: https://github.com/search?q=is%3Apr+org%3Arust-lang+is%3Amerged+merged%3A2019-11-18..2019-11-25
+
+* [stabilize `!`](https://github.com/rust-lang/rust/pull/65355)
+* [stabilize `cfg(doc)`](https://github.com/rust-lang/rust/pull/61351)
+* [debuginfo: support for `std::collections::Hash*` in windows debuggers](https://github.com/rust-lang/rust/pull/66597)
+* [make gdb pretty-printing more robust when printing uninitialized `Vec`](https://github.com/rust-lang/rust/pull/66576)
+* [generate DWARF address ranges for faster lookups](https://github.com/rust-lang/rust/pull/66532)
+* [fix cycle when debug-printing opaque types](https://github.com/rust-lang/rust/pull/66594)
+* [resolve: give derive helpers highest priority during resolution](https://github.com/rust-lang/rust/pull/66529)
+* [remove pretty printing of specific nodes in AST](https://github.com/rust-lang/rust/pull/66575)
+* [point at type in `let` assignment on type errors](https://github.com/rust-lang/rust/pull/66539)
+* [suggest calling async closure when needed](https://github.com/rust-lang/rust/pull/66239)
+* [suggest `#[repr(C)]` instead of `#[repr(C, packed, ...)]`](https://github.com/rust-lang/rust/pull/66206)
+* [add outlives suggestions for some lifetime errors](https://github.com/rust-lang/rust/pull/58281)
+* [use a `SmallVec` for `Candidate::match_pairs`](https://github.com/rust-lang/rust/pull/66540)
+* [miri: add `acos`, `asin`, and `atan` foreign functions](https://github.com/rust-lang/miri/pull/1067)
+* [mir-opt: asking `?`s in a more optimized fashion](https://github.com/rust-lang/rust/pull/66282)
+* [mir-opt: turn on the `ConstProp` pass by default](https://github.com/rust-lang/rust/pull/66074)
+* [miri: support unwinding after a panic](https://github.com/rust-lang/miri/pull/693)
+* [handle statics in MIR as const pointers](https://github.com/rust-lang/rust/pull/66587)
+* [delay an `is_local_ever_initialized` call](https://github.com/rust-lang/rust/pull/66537)
+* [reduce size of `hir::Expr` by boxing more of `hir::InlineAsm`](https://github.com/rust-lang/rust/pull/66515) 
+* [use proc-macro to derive HashStable everywhere](https://github.com/rust-lang/rust/pull/66279)
+* [remove `compiler_builtins_lib` feature from libstd](https://github.com/rust-lang/rust/pull/66538)
+* [std::error::Chain: remove `Copy`](https://github.com/rust-lang/rust/pull/66511)
+* [use `drop_in_place` in `array::IntoIter::drop`](https://github.com/rust-lang/rust/pull/65821)
+* [stabilize `Result::map_or_else`](https://github.com/rust-lang/rust/pull/66322)
+* [libc: deprecate vfork](https://github.com/rust-lang/libc/pull/1574)
+* [libc: add initial support for sparc-unknown-linux-gnu](https://github.com/rust-lang/libc/pull/1567)
+* [cargo: extend documentation on security concerns of crate names in a registry](xtend documentation on security concerns of crate names in a registry)
```

Comment:
Shouldn't this be a link to a valid url?

Reply:
fixed, thanks for pointing it out.